pkg/virt-controller/watch/vmi.go

```diff
@@ -1396,6 +1396,7 @@ func (c *VMIController) triggerHotplugPopulation(volume *virtv1.Volume, vmi *vir
 			return &syncErrorImpl{fmt.Errorf("Error creating hotplug population trigger pod %v", err), FailedCreatePodReason}
 		}
 	}
+	c.recorder.Eventf(vmi, k8sv1.EventTypeNormal, SuccessfulCreatePodReason, "Created hotplug trigger pod for volume %s", volume.Name)
```

Comment:
shouldn't this go into the if statement right about this? that way we're sure we only get this event once per a pod creation?

Reply:
Good point, fixed.